docs/getting-started.rst

```diff
@@ -0,0 +1,98 @@
+Getting Started
+===============
+
+Benchmarks are `Juju Actions <https://jujucharms.com/docs/stable/actions>`_ that follow a specific pattern. While they can be accomplished via ``action_set``, we've created charm-benchmark to make that task easier.
+
+Installing Charm Benchmark
+--------------------------
+
+    pip install charm-benchmark
+
+
+Using charm-benchmark
+---------------------
+
+There are two ways of using charm-benchmark: via direct calls to the Python library...:
+
+::
+
+    #!/usr/bin/env python
+    import subprocess
+
+    try:
+        from charmhelpers.core.hookenv import action_get
```

Comment:
Why do we need this? Since charmhelpers is a dep of charmbenchmark, importing charmbenchmark should catch this?